GetIntoTeachingApi/Attributes/CrmETagAttribute.cs

```diff
@@ -14,22 +15,26 @@ public class CrmETagAttribute : Attribute, IActionFilter
     {
         private readonly JobStorage _hangfireJobStorage;
         private readonly IMetricService _metrics;
+        private readonly IEnv _env;
 
         public CrmETagAttribute()
         {
             _metrics = new MetricService();
             _hangfireJobStorage = JobStorage.Current;
+            _env = new Env();
         }
 
-        public CrmETagAttribute(JobStorage hangfireJobStorage, IMetricService metrics)
+        public CrmETagAttribute(JobStorage hangfireJobStorage, IMetricService metrics, IEnv env)
         {
             _hangfireJobStorage = hangfireJobStorage;
             _metrics = metrics;
+            _env = env;
         }
 
         public void OnActionExecuting(ActionExecutingContext context)
         {
-            var canCacheRequest = context.HttpContext.Request.Method == "GET";
+            var isGetRequest = context.HttpContext.Request.Method == "GET";
+            var canCacheRequest = isGetRequest && !_env.IsDevelopment;

```

Comment:
Does this need to handle test environments as well? Presumably caching should be disabled in those?

Reply:
Yeah that's a good shout; at the moment I mock it to disable caching but that would be better as the default behaviour - I'll update this in a bit